packages/compass-saved-aggregations-queries/src/components/aggregations-queries-list.tsx

```diff
@@ -1,15 +1,37 @@
-import React from 'react';
-import { connect } from 'react-redux';
+import React, { useEffect } from 'react';
+import { connect, ConnectedProps } from 'react-redux';
+import type { State } from './../stores/aggregations-queries-items';
+import { fetchItems } from './../actions/aggregations-queries-actions';
 
-const AggregationsQueriesList: React.FunctionComponent = () => {
-  return <div>Hello, world!</div>;
+const AggregationsQueriesList = ({
+  loading,
+  items,
+  fetchItems,
+}: PropsFromRedux) => {
+  useEffect(() => {
+    fetchItems();
+  }, [fetchItems]);
+
+  if (loading) {
+    return <p>Loading ...</p>;
+  }
+  if (!loading && !items.length) {
```

Comment:
If loading is already returning

```suggestion
  if (!items.length) {
```